### DISCUSSION TITLE
【322330】饥荒；联机版-本体

### DISCUSSION THREAD

**Discussion by XiChen03** [pjy612/SteamManifestCache#229] [Share]
appid：322330
Key:
```
322331;3ee94d4c47554fc580de37cdeb4bb41020ece2d5ab195d51d8b87f5e438e9f23
```

Code：74CD5329137EA99B
Email：2639943898@qq.com

**Comment by pjy612** [comment]
仅在线联机的没用

**Reply by XiChen03** [reply]
啧 我发的俩都没用啊 我再重新编辑个发发

**Reply by XiChen03** [reply]
这个饥荒；联机版应该行了吧

**Reply by pjy612** [reply]
查收

**Reply by aipave** [reply]
怎么拿到自己的code: xxxx

**Reply by Niguyt5874** [reply]
运行一下软件会自动生成一个code

**Comment by aipave** [comment]
是cmd界面的会有一个code 吗, 我用everything搜code.txt, 全局没有看到
________________________________
发件人: Niguyt5874 ***@***.***>
发送时间: 2024年1月6日 19:07
收件人: pjy612/SteamManifestCache ***@***.***>
抄送: PeiChen Lyu ***@***.***>; Comment ***@***.***>
主题: Re: [pjy612/SteamManifestCache] 【322330】饥荒；联机版-本体 (Discussion #229)


运行一下软件会自动生成一个code

―

**Reply by aipave** [reply]
生成的目录在哪里, 没找到, 大佬求教

**Comment by aipave** [comment]
大佬, 感谢回复, 我没看到
我运行了之后进入的cmd界面输出:
```
######################################## 10.01MiB/10.01MiB
2023-12-26 修复最新测试版无法解锁的问题！
2024-01-01 更新GUI，支持高DPI，更友好的提升用户体验（由于临时赶工可能出现一些未知问题，欢迎及时反馈。右下角托盘图标反馈 ）！
2024-01-01 如果原有游戏解锁失效请重新拖拽或按住Ctrl拖拽移除后重新拖拽入库！
2024-01-01 优化拖拽支持（文本，链接，Steam游戏库，Steam生成的快捷方式，已编译的脚本.st，脚本.lua）！
2024-01-02 修复按住Ctrl拖拽无法移除appid的问题！
2024-01-05 修复已知BUG，更改Lua规则，优化授权操作！
2024-01-05 新增成就解锁，成就导入和导出以及查看！
2024-01-05 新增设置 Steam 的启动参数！
2024-01-05 新增禁止游戏更新，仅禁止已解锁的游戏列表，开启禁止更新后所有已解锁的游戏将无内容（优点是可永久阻止游戏的更新 防止更新后无法进入游戏）！
2024-01-06 修复Lua中的解析错误导致异常崩溃！
按回车键退出...
```

[cid:27c0ead8-c1e4-4dd1-adb5-03c49b535bee]


________________________________
发件人: Niguyt5874 ***@***.***>
发送时间: 2024年1月6日 19:07
收件人: pjy612/SteamManifestCache ***@***.***>
抄送: PeiChen Lyu ***@***.***>; Comment ***@***.***>
主题: Re: [pjy612/SteamManifestCache] 【322330】饥荒；联机版-本体 (Discussion #229)


运行一下软件会自动生成一个code

―

**Comment by aipave** [comment]
我运行了之后进入的cmd界面输出:
```
######################################## 10.01MiB/10.01MiB
2023-12-26 修复最新测试版无法解锁的问题！
2024-01-01 更新GUI，支持高DPI，更友好的提升用户体验（由于临时赶工可能出现一些未知问题，欢迎及时反馈。右下角托盘图标反馈 ）！
2024-01-01 如果原有游戏解锁失效请重新拖拽或按住Ctrl拖拽移除后重新拖拽入库！
2024-01-01 优化拖拽支持（文本，链接，Steam游戏库，Steam生成的快捷方式，已编译的脚本.st，脚本.lua）！
2024-01-02 修复按住Ctrl拖拽无法移除appid的问题！
2024-01-05 修复已知BUG，更改Lua规则，优化授权操作！
2024-01-05 新增成就解锁，成就导入和导出以及查看！
2024-01-05 新增设置 Steam 的启动参数！
2024-01-05 新增禁止游戏更新，仅禁止已解锁的游戏列表，开启禁止更新后所有已解锁的游戏将无内容（优点是可永久阻止游戏的更新 防止更新后无法进入游戏）！
2024-01-06 修复Lua中的解析错误导致异常崩溃！
按回车键退出...
```
大佬, 感谢回复, 我没看到
[cid:27c0ead8-c1e4-4dd1-adb5-03c49b535bee]


________________________________
发件人: Niguyt5874 ***@***.***>
发送时间: 2024年1月6日 19:07
收件人: pjy612/SteamManifestCache ***@***.***>
抄送: PeiChen Lyu ***@***.***>; Comment ***@***.***>
主题: Re: [pjy612/SteamManifestCache] 【322330】饥荒；联机版-本体 (Discussion #229)


运行一下软件会自动生成一个code

―

**Reply by Niguyt5874** [reply]
你要随便入库一个游戏就会自动生成code的文本

**Comment by Niguyt5874** [comment]
比如入库1958220这个游戏&nbsp; 入库完后就会自动生成一个code文件&nbsp; 就在本目录里






------------------&nbsp;原始邮件&nbsp;------------------
发件人: "PeiChen ***@***.***&gt;; 
发送时间: 2024年1月6日(星期六) 晚上7:40
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [pjy612/SteamManifestCache] 【322330】饥荒；联机版-本体 (Discussion #229)





 
生成的目录在哪里, 没找到, 大佬求教

**Comment by Niguyt5874** [comment]
别忘了 是用sai入库&nbsp; &nbsp;刚才打游戏没注意邮箱 不好意思了




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "pjy612/SteamManifestCache"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月6日(星期六) 晚上7:40
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [pjy612/SteamManifestCache] 【322330】饥荒；联机版-本体 (Discussion #229)





 
生成的目录在哪里, 没找到, 大佬求教

**Comment by aipave** [comment]
大佬, 哪个sai是什么, 和streamtools不是同一个东西吧
________________________________
发件人: Niguyt5874 ***@***.***>
发送时间: 2024年1月6日 20:25
收件人: pjy612/SteamManifestCache ***@***.***>
抄送: PeiChen Lyu ***@***.***>; Comment ***@***.***>
主题: Re: [pjy612/SteamManifestCache] 【322330】饥荒；联机版-本体 (Discussion #229)


别忘了 是用sai入库&nbsp; &nbsp;刚才打游戏没注意邮箱 不好意思了




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "pjy612/SteamManifestCache" ***@***.***&gt;;
发送时间:&nbsp;2024年1月6日(星期六) 晚上7:40
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [pjy612/SteamManifestCache] 【322330】饥荒；联机版-本体 (Discussion #229)






生成的目录在哪里, 没找到, 大佬求教

―
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***&gt;

―

**Reply by Niguyt5874** [reply]
就你用的那个软件啊

**Reply by aipave** [reply]
我用的软件名是这个 SteamtoolsSetup.exe

**Comment by Niguyt5874** [comment]


**Reply by aipave** [reply]
看到了 SteamAppImport
